egs/swbd/s5c/local/reverberate/IR_stats.m

```diff
@@ -0,0 +1,348 @@
+function [rt,drr,cte,cfs,edt] = IR_stats(x, fs, varargin)
+%function [rt,drr,cte,cfs,edt] = IR_stats(filename, varargin)
```

Comment:
I don't see any point distributing scripts that have a dependency on code that we can't distribute.  This would only work on our grid -> pointless to check in, surely.